integration/network/macvlan/macvlan_driver.bats

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bats
+# *-*- Mode: sh; sh-basic-offset: 8; indent-tabs-mode: nil -*-*
```

Comment:
I only see this line in 3 other bats files - do we @kata-containers/tests have any preferences here?
We don't have such lines in any other files afaik, so unless there is a really strong 'it is necessary to be able to read the file at all' argument, then my vote is probably against.